client/import.go

```diff
@@ -202,6 +213,10 @@ func (c *Client) Import(ctx context.Context, reader io.Reader, opts ...ImportOpt
 					Target: m,
 				})
 			}
+			// don't create images for referrer descriptors
+			if _, ok := m.Annotations["org.opencontainers.image.referrer.subject"]; ok {
```

Comment:
@sudo-bmitch this seems based on your PR https://github.com/opencontainers/image-spec/pull/1171, can you take a look at how it is used here?